lib/index.js

```diff
@@ -53,12 +59,27 @@ function handle(room, peerToPeer, AppID, AppSecretOrTokenGenerator, localUserID,
 
   // Create and initialize the callstats object.
 
-  let callstats = callstatsModule();
+  callstats = callstatsModule();
 
   callstats.initialize(AppID, AppSecretOrTokenGenerator, localUserID, csInitCallback, csStatsCallback, configParams);
+  if (room) {
+    let sessionHandler = new SessionHandler(room, peerToPeer, room.name, callstats);
+    room.callstatsSessionHandler = sessionHandler;
```

Comment:
could you just call the setTwilioVideoRoom function instead of duplicating the code here?